DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -286,17 +296,20 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
             $definition->setFactoryMethod('login');
         }
 
+        $workspace = $admin ? 'default' : $session['workspace'];
```

Comment:
i think we should not hardcode this but have a configuration option for admin_workspace that defaults to 'default'.